src/ApiService/ApiService/Program.cs

```diff
@@ -47,13 +48,22 @@ public class LoggingMiddleware : IFunctionsWorkerMiddleware {
 
         using var host =
             new HostBuilder()
+            .ConfigureAppConfiguration(builder => {
+                var _ = builder.AddAzureAppConfiguration(options => {
+                    var _ = options.Connect(new Uri(configuration.AppConfigurationEndpoint!), new ManagedIdentityCredential())
+                        .ConfigureRefresh(refreshOptions =>
+                            refreshOptions.SetCacheExpiration(TimeSpan.FromMinutes(1)));
+                });
+            })
             .ConfigureFunctionsWorkerDefaults(builder => {
                 builder.UseMiddleware<LoggingMiddleware>();
                 builder.AddApplicationInsights(options => {
                     options.ConnectionString = $"InstrumentationKey={configuration.ApplicationInsightsInstrumentationKey}";
                 });
+                builder.UseAzureAppConfiguration();
             })
             .ConfigureServices((context, services) => {
+                services.AddAzureAppConfiguration();
```

Comment:
Not sure if you found this while enabling AppConfiguration but with the AppConfiguration stuff there's also [explicit feature flag support](https://learn.microsoft.com/en-us/azure/azure-app-configuration/quickstart-feature-flag-azure-functions-csharp?tabs=in-process) via `AddFeatureManagement` which gives you an `IFeatureManagerSnapshot` injected to use instead of having to parse the configuration settings manually out of e.g. `context.AppConfiguration[FeatureFlagConstants.EnableJinjaConvert]` (instead it would be something like `_features.IsEnabledAsync(FeatureFlagConstants.EnableJinjaConvert)`).